src/translations/pl/translation.json

```diff
@@ -444,46 +444,46 @@
     "Not wearing:": "Nie nosząc:",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_many": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}}": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}}",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_many": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}}": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}}",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
```

Comment:
The translation of the seconds for `seconds_one` shouldn't be `sekundę`?

Eg:

> While having the **pain** effect(s) on your **arm** for **1** second

p.s. this comment applies to other strings too where you see the `seconds_one` quantifier

Reply:
Thanks! I didn't realize there are one/few/many, that should solve the plurals for Polish, though I bet there will be some fixes afterwards :)